index.js

```diff
@@ -82,9 +82,15 @@ function fetchPageOfDataAndFilter(url) {
                     })
                     .catch((err) => {
                         console.log('Error turning response into JSON:', err);
+
+                        return reject(err);
```

Comment:
I'm curious about the choice of calling reject here, rather than re-throwing the error. 

Also, is rejecting the right thing to do here, as opposed to logging and continuing to process other users? I think that's a requirements question for Dani.

Reply:
Good question. Yes, I think the correct thing is to not rethrow the error, so the code is handling it correctly.